test/oz/erc-4626/TokenVault.js

```diff
@@ -0,0 +1,92 @@
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+const Constants = require('../../constants')
+
+describe("@OZ TokenVault Contract", function () {
+  let TokenVault;
+  let tokenVault;
+  let ERC20Mock;
+  let asset;
+  let owner;
+  let addr1;
+  let addr2;
+  let addrs;
+
+  beforeEach(async function () {
+    ERC20Mock = await ethers.getContractFactory("contracts/erc-20/ERC20Mock.sol:ERC20Mock");
+    asset = await ERC20Mock.deploy("MockToken", "MTK", Constants.GAS_LIMIT_1_000_000);
```

Comment:
Please make "MockToken" and "MTK" constant variables as I see them being reused around in the contract